applications/Unity.GrantManager/src/Unity.GrantManager.Web/Views/Shared/Components/ProjectInfo/Default.cshtml

```diff
@@ -141,15 +141,31 @@
                     </abp-select>
                 </abp-column>
             </abp-row>
-          
+         </abp-row>
+        <abp-row class="m-0 p-0">
+            <abp-column size="_12" class="d-inline-flex justify-content-between pb-3 pt-2 project-location">
+                <div id="ProjectInfoTitleRegion" class="d-flex align-items-stretch justify-content-start">
+                    <h6 class="ps-1 fw-bold">@L["ProjectInfoView:ProjectInfoContract"].Value</h6>
+                </div>
+            </abp-column>
+        </abp-row>
+        <abp-row class="m-0 project-info-location">
+            <abp-row class="m-0 p-0">
+                <abp-column size="_4" class="px-1">
+                    <abp-input asp-for="@Model.ProjectInfo.ContractNumber" onchange="enableProjectInfoSaveBtn(this)" />
```

Comment:
should we not make this type="number"

Reply:
> should we not make this type="number"

hi @AndreGAot -- i have now made the necessary changes.